.ncurc.yaml

```diff
@@ -3,4 +3,5 @@ reject: [
   # TODO: v6+ (ref: https://github.com/sindresorhus/get-port/releases/tag/v6.0.0) is an ESM library and thus not compatible with CommonJS. Future story needed to resolve.
   "get-port",
   "@mojaloop/sdk-standard-components",
+  "axios"
```

Comment:
Please mention the reason for disabling this. As you mentioned in the chat "The reason I excluded is that after updating it to the latest version (from 1.6.7 to 1.6.8 ), I can't mock it in unit-tests any more."

Reply:
done